test.ts

```diff
@@ -1,10 +1,7 @@
-import {
-  assert,
-  assertEquals,
-  assertIsError,
-} from "https://deno.land/std@0.200.0/assert/mod.ts";
+import { assert, assertEquals, assertIsError } from "jsr:@std/assert@1.0.5";
```

Comment:
```suggestion
import { assert, assertEquals, assertIsError } from "jsr:@std/assert@^1";
```
Let's be lax with test requirements